cura/CuraActions.py

```diff
@@ -73,6 +75,36 @@ def centerSelection(self) -> None:
             operation.addOperation(center_operation)
         operation.push()
 
+    # Rotate the selection, so that the face that the mouse-pointer is on, faces the build-plate.
+    @pyqtSlot()
+    def bottomFaceSelection(self) -> None:
+        selected_face = Selection.getSelectedFace()
+        if not selected_face:
+            Logger.log("e", "Bottom face operation shouldn't have been called without a selected face.")
+            return
+
+        original_node, face_id = selected_face
+        meshdata = original_node.getMeshDataTransformed()
+        if not meshdata or face_id < 0 or face_id > 0x10001:
```

Comment:
I'm not familiar with this part, so perhaps document what `face_id < 0` and `face_id >0x10001` mean? Those are magic numbers to me.

Reply:
Fixed!